crates/ty/tests/cli.rs

```diff
@@ -447,12 +447,12 @@ fn cli_rule_severity() -> anyhow::Result<()> {
         for a in range(0, int(y)):
             x = a
 
-        print(x)  # possibly-unresolved-reference
+        prin(x)  # unresolved-reference
         "#,
     )?;
 
-    // Assert that there's a possibly unresolved reference diagnostic
-    // and that division-by-zero has a severity of error by default.
+    // Assert that there's no unresolved reference diagnostic by default
+    // but a division-by-zero (error) and a unresolved-import (error) diagnostic
```

Comment:
Similar here, this change looks wrong? It should refer to the first snapshot assertion, not the second-